docs/package.json

```diff
@@ -14,36 +14,36 @@
   "dependencies": {
     "@algolia/autocomplete-core": "^1.7.3",
     "@algolia/autocomplete-preset-algolia": "^1.7.3",
+    "@dust-tt/sparkle": "0.1.26",
     "@headlessui/react": "^1.7.7",
     "@mdx-js/loader": "^2.3.0",
     "@mdx-js/react": "^2.3.0",
-    "@next/mdx": "^13.5.4",
+    "@next/mdx": "^14.2.3",
     "@sindresorhus/slugify": "^2.1.1",
     "@tailwindcss/typography": "^0.5.8",
     "acorn": "^8.8.1",
     "algoliasearch": "^4.14.2",
     "autoprefixer": "^10.4.12",
     "clsx": "^1.2.0",
     "dd-trace": "^5.1.0",
-    "@dust-tt/sparkle": "0.1.26",
     "focus-visible": "^5.2.0",
     "framer-motion": "7.8.1",
     "mdast-util-to-string": "^3.1.0",
     "mdx-annotations": "^0.1.1",
-    "next": "^13.5.4",
+    "next": "^14.2.3",
     "postcss-focus-visible": "^6.0.4",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "react": "^18",
```

Comment:
IMHO those dependencies should be stricter.

Reply:
Why?